invenio_rdm_records/services/schemas/__init__.py

```diff
@@ -104,6 +105,13 @@ def default_nested(self, data, many, **kwargs):
 
         return data
 
+    @pre_load()
+    def remove_status(self, data, many, **kwargs):
+        """Remove access status if present in incoming serialized object."""
+        if data.get('access', {}).get('status'):
+            data.get('access', {}).pop('status')
+        return data
```

Comment:
@zzacharo I had to add this, otherwise, the UI would send back the `access.status` and a validation error would occur with no message. The service layer schema fails as there is no field `access.status`. Any idea how to better tackle it?

Reply:
I think the UI should handle the pop of this data....the REST API is fine I think to fail with validation error if the status is passed as it's dump_only field... Now, in the UI we have a serializer class (See https://github.com/inveniosoftware/react-invenio-deposit/blob/master/src/lib/DepositRecordSerializer.js#L86). You need to create a class e.g `AccessField` and control the (de)serialization for the [access](https://github.com/inveniosoftware/react-invenio-deposit/blob/master/src/lib/DepositRecordSerializer.js#L59) field!